cmd/showconfig.go

```diff
@@ -49,13 +49,14 @@ func showConfig(
 	if err != nil {
 		panic(err)
 	}
-	out, err := yaml.Marshal(cfgMap)
-	if err != nil {
+
+	encoder := yaml.NewEncoder(outputter)
+	encoder.SetIndent(2)
+	if err := encoder.Encode(cfgMap); err != nil {
 		return stackerr.NewStackErrf(err, "failed to marshal yaml")
 	}
 
 	log.Info().Msgf("Using config: %s", config.Config)
 
-	fmt.Fprintf(outputter, "%s", string(out))
```

Comment:
Because it's not needed anymore.

`encoder` writes directly to `outputter` (`encoder := yaml.NewEncoder(outputter)`).

`TestShowCfg` proofs that this refactoring doesn't break output.

Reply:
Ah! Wonderful, thank you for the cleanup.